tenseal/tensors/__init__.py

```diff
@@ -120,7 +122,7 @@ def ckks_vector_from(context, data):
     )
 
 
-def ckks_tensor(context, tensor, scale=None):
+def ckks_tensor(context, tensor, scale=None, batch=True):
```

Comment:
I think beginner users might not expect the batching to happen, they might find it weird somehow. Can we make the default to false here?